### PR TITLE
Fix: Powder cost formulas and calculations

### PR DESCRIPTION
**Pull request by lunaynx**
## What
Fixed a bunch of errors and inconsistencies in the HotmAPI powder cost formulas, as well as made the powder spent calculation more logical.

## Changelog Fixes
+ Fix incorrect powder costs and powder spent amounts being displayed for some HOTM perks. - Luna

### CODE REVIEW

File: src/main/java/at/hannibal2/skyhanni/data/HotmData.kt [hunk 1]
```diff
@@ -57,50 +56,50 @@ private fun calculatePeakOfTheMountainLoot(level: Int): Map<HotmReward, Double>
...
+        { level -> (level + 1.0).pow(3.05).toInt() },
         { level -> mapOf(HotmReward.MINING_FORTUNE to level * 5.0) },
     ),
     QUICK_FORGE(
         "Quick Forge",
         20,
-        { currentLevel -> (currentLevel + 2.0).pow(4) },
-        { level -> mapOf(HotmReward.FORGE_TIME_DECREASE to 10.0 + (level * 0.5)) },
+        { level -> (level + 1.0).pow(4).toInt() },
+        { level -> mapOf(HotmReward.FORGE_TIME_DECREASE to if (level >= 20) 30.0 else 10.0 + (level * 0.5)) },
```
Comment (Thunderblade73) [inline src/main/java/at/hannibal2/skyhanni/data/HotmData.kt:79]: are you sure that it does not work with blue egg?

Comment (Thunderblade73) [reply src/main/java/at/hannibal2/skyhanni/data/HotmData.kt:79]: oh, the updated the wiki formula
```
Reward Formula: min(30, 10+(Level*0.5)+floor(Level/20)*20)
```

Comment (lunaynx) [reply src/main/java/at/hannibal2/skyhanni/data/HotmData.kt:79]: Yeah, I'm pretty sure blue egg doesn't buff it further, but I can have someone confirm.

Comment (lunaynx) [reply src/main/java/at/hannibal2/skyhanni/data/HotmData.kt:79]: Confirmed.
image

**Pull request merged**